src/components/Plugin/Hal/CustomBlock.vue

```diff
@@ -0,0 +1,55 @@
+<script setup>
+const props = defineProps({
+  space: Object
+});
+
+const getSpaceId = () => {
```

Comment:
You should be using `computed` here like this.
```
const spaceId = computed(() => props.space.id)
```
But in this case you can just use `props.space.id` inside script and `space.id` inside template.